security/ssh_certificate.ksy

```diff
@@ -0,0 +1,196 @@
+meta:
+  id: ssh_certificate
+  title: OpenSSH Certificate
+  license: CC0-1.0
+  endian: be
+  imports:
+    - /security/ssh_public_key
```

Comment:
`imports` should be above `endian` (https://doc.kaitai.io/ksy_style_guide.html#meta), but that's just a nitpick:

```suggestion
  imports:
    - /security/ssh_public_key
  endian: be
```